python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -392,6 +392,13 @@ keys:
             - str
           min: 0
           max: 300
+        min_links:
+          type: int
+          description: Minimum no of ports required up before bringing up a port-channel.
+          convert_types:
+            - str
+          min: 0
+          max: 16
```

Comment:
i can see the different range here in my latest EOS 4.33F. Please check if this very as per hardware or EOS version.
wan1-dc1(config-if-Po1)#port-channel min-links ?
  <0-64>  Minimum no of ports required up before bringing up a port-channel

Reply:
I think this is highly platform dependent, so let's use the higher number.